tofu/tests/tests01_geom/tests01_GG.py

```diff
@@ -113,16 +113,23 @@ def test02_Poly_CLockOrder():
 
     # Test arbitrary 2D polygon
     Poly = np.array([[0.,1.,1.,0.],[0.,0.,1.,1.]])
+
     P = GG.format_poly(Poly, order='C', Clock=False, close=True,
                        Test=True)
+
     assert all([np.allclose(P[:, 0], P[:, -1]), P.shape == (2, 5),
                 not GG.Poly_isClockwise(P), P.flags['C_CONTIGUOUS'],
                 not P.flags['F_CONTIGUOUS']])
+
     P = GG.format_poly(Poly, order='F', Clock=True, close=False,
                        Test=True)
-    assert all([not np.allclose(P[:, 0], P[:, -1]), P.shape == (2, 4),
-                GG.Poly_isClockwise(np.concatenate((P, P[:, 0:1]), axis=1)),
-                not P.flags['C_CONTIGUOUS'], P.flags['F_CONTIGUOUS']])
+
+    assert not np.allclose(P[:, 0], P[:, -1]), "poly should not be closed"
+    assert P.shape == (2, 4), ("shape of poly should be (2,4), here = "
+                               + str(P.shape) + "\n Poly = " + str(P) )
+    assert GG.Poly_isClockwise(np.concatenate((P, P[:, 0:1]), axis=1))
+    assert not P.flags['C_CONTIGUOUS']
+    assert P.flags['F_CONTIGUOUS']
```

Comment:
In addition to this,
could you implement just a single more unit test to do the computation of volume sampling with the old and the new algorithm and make sure they return the same result ?

Reply:
yes